src/desktop/assets/viewing-room.tsx

```diff
@@ -0,0 +1 @@
+import "desktop/apps/viewing-room/client"
```

Comment:
What's the thinking with continuing to split out separate 'apps' for the routes? Is it just because we don't have proper route bundle splitting yet?

Reply:
Once the new app shell A/B test is merged we shouldn't have to do this anymore. We have bundle splitting in place. For users in the control there still needs to be a separate place to mount this client code, though. 

(Once it launches all of these individual apps / mount points / etc can be deleted from the code base.)

Reply:
Ah, I see. Nice.